recipes/libraw/all/conanfile.py

```diff
@@ -1,17 +1,19 @@
-from conans import ConanFile, tools, CMake
+from conan import ConanFile
+from conan.tools.files import get, copy, collect_libs
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
 import os
 
+required_conan_version = ">=1.53.0"
+
 class LibRawConan(ConanFile):
     name = "libraw"
     description = "LibRaw is a library for reading RAW files obtained from digital photo cameras (CRW/CR2, NEF, RAF, DNG, and others)."
-    topics = ["image", "photography", "raw"]
+    license = "CDDL-1.0/LGPL-2.1-only"
```

Comment:
Should these be separated out?

```suggestion
    license = "CDDL-1.0", "LGPL-2.1-only"
```